.circleci/config.yml

```diff
@@ -163,7 +186,8 @@ jobs:
 
   build_and_test:
     <<: *test_container_config
-    resource_class: small
+    <<: *defaults
+    resource_class: xlarge
```

Comment:
The default resource_class can be used, it doesn't need to be xlarge. (It is on CT because it has so many tests).
This line can be removed.